flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -705,7 +705,7 @@ public TableResult executeInternal(QueryOperation operation) {
 					.tableSchema(operation.getTableSchema())
 					.data(resultProvider.getResultIterator())
 					.setPrintStyle(TableResultImpl.PrintStyle.tableau(
-							PrintUtils.MAX_COLUMN_WIDTH, PrintUtils.NULL_COLUMN, isStreamingMode))
+							PrintUtils.MAX_COLUMN_WIDTH, PrintUtils.NULL_COLUMN, true, isStreamingMode))
```

Comment:
Can we use `isStreamingMode` for the `deriveColumnWidthByType` instead of hard code to `true`? 

In my understanding, we can derive width by content if it is batch mode.

Reply:
The batch select result may be very large to be stored in local java heap memory, so I prefer to derive column type for both mode select result.